node/modules/storageminer.go

```diff
@@ -351,6 +352,44 @@ func SectorStorage(mctx helpers.MetricsCtx, lc fx.Lifecycle, lstor *paths.Local,
 	return sst, nil
 }
 
+func F3Participation(mctx helpers.MetricsCtx, lc fx.Lifecycle, api v1api.FullNode, minerAddress dtypes.MinerAddress) error {
```

Comment:
why is this needed in miner?

Reply:
Keys and libp2p are in Lotus, but Lotus itself has no idea what miners are running that are using it.
So this is the miner telling Lotus: I want you to participate in the F3 network for me.

Reply:
got it